v_3_0_0/fake_extension.go

```diff
@@ -1,6 +1,11 @@
 package v_3_0_0
 
-type FakeExtension struct{}
+type FakeKeys struct {
+	APIServerEncryptionKey []byte
+}
+type FakeExtension struct {
+	Keys FakeKeys
+}
```

Comment:
I think it should be added to Params struct.

Reply:
It's part of Extension, but the extension will be overwritten

Reply:
Do you prefer to have it, as new Param field?

Reply:
FakeExtensions is only for testing purposes. Extension interface is here https://github.com/giantswarm/k8scloudconfig/blob/d75265767b2a885d6cfb004ff00ab700908b8d03/v_3_0_0/types.go#L95 and I don't see a reason why we would add a new field (or rather method - it's an interface) to it.